README.md

```diff
@@ -20,7 +20,7 @@
 * Designed with modern versions of macOS (10.11+) in mind
 * All the features you need for video and music: subtitles, playlists, chapters…and much, much more!
 * Force Touch, picture-in-picture and advanced Touch Bar support
-* Customizable user interface including multiple color schemes and on screen controller (OSC) layout positioning
+* Customizable user interface including multiple color schemes and on-screen controller (OSC) layout positioning
```

Comment:
```suggestion
* Customizable user interface including multiple color schemes and on screen controller (OSC) layout positioning
```

IINA uses "on screen controller"

Reply:
It's just that most dictionaries just stick to the spelling that I suggested ([reference 1](https://en.wiktionary.org/wiki/on-screen), [reference 2](https://www.oxfordlearnersdictionaries.com/definition/english/on-screen_1), [reference 3](https://www.merriam-webster.com/dictionary/on-screen), [reference 4](https://www.britannica.com/dictionary/on%E2%80%93screen)).

Reply:
Please keep this as-is until we decide to change this for IINA as a whole

Reply:
Done.